debian/NOTICE

```diff
@@ -0,0 +1,99 @@
+NOTICES AND INFORMATION
```

Comment:
If we need to add the NOTICE file to the citus repo, can we use that file and _not_ create another copy in here?

Reply:
Actually I don't like the idea. I understand you are trying to enforce DRY principle but it is not a good practice to get an online file during packaging process which may cause flakiness.

Reply:
Each version of Citus can potentially have different NOTICE files. How do you plan to maintain different versions of these files?

I do not think that we can move forward without having a working plan for this problem.

Reply:
I went through all Citus versions >= 9.2 and the NOTICE file is correct for all of those.

I am ok with keeping the NOTICE file here, for now.